pkg/consts/consts.go

```diff
@@ -0,0 +1,20 @@
+package consts
+
+const (
+	// Listening port for the OCM Agent web service
+	OCMAgentServicePort = 8081
+	// Listening port for the OCM Agent metrics
+	OCMAgentMetricsPort = 8383
+
+	// Metrics path for OCM Agent service
+	MetricsPath = "/metrics"
+	// Ready probe path for OCM Agent web service
+	ReadyzPath = "/readyz"
+	// Live probe path for OCM Agent web service
+	LivezPath = "/livez"
+	// Alertmanger webhook receiver path
+	WebhookReceiverPath = "/alertmanager-receiver"
+
+	// Service name for the sending service logs
+	ServiceLogServiceName = "SREManualAction"
```

Comment:
For an automated Service Log sent via OCM Agent, should the value still be `SREManualAction`? Not sure if there are other values possible for this or not from OCM side.

Reply:
yeah, I have the same concern, but I did not change it (just moved it from other place to here.)

One thing to think together. The `osdctl servicelog list` is hardcoded to search SLs with the service_name = SREManualAction 
https://github.com/openshift/osdctl/blob/e152fe0698d1ea62c9094abdb48c9021e3d1cde1/cmd/servicelog/list.go#L94

Reply:
This will probably eventually change as a result of the SDB ticket. It's fine to stay as is for now.

Reply:
Ack.. Resolving comment for now